sqs/runtime/src/main/java/io/quarkus/amazon/sqs/runtime/SqsDevServicesBuildTimeConfig.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.amazon.sqs.runtime;
+
+import java.util.Set;
+
+import io.quarkus.amazon.common.runtime.DevServicesBuildTimeConfig;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class SqsDevServicesBuildTimeConfig extends DevServicesBuildTimeConfig {
+
+    /**
+     * The queues to create on startup.
+     */
+    @ConfigItem(defaultValue = "default")
```

Comment:
So you will always create a `default` queue?

Reply:
I was following what was done with S3... So yes, we always create a `default` queue as we have a `default` S3 bucket. But I can change this if you want.

Reply:
I guess it's a mistake in s3 too. There is no link between this value and the actual bucket used by the client. So a bucket is created but in most cases it is not used.

Reply:
I don't create a queue by default now.